cronjob-source.yml

```diff
@@ -1,11 +0,0 @@
-apiVersion: sources.eventing.knative.dev/v1alpha1
```

Comment:
Is there a reason why this file was deleted?

Reply:
We aren't using this file. It uses a deprecated 'kind' of "CronJobSource", which has been replaced by "PingSource", as of Knative v0.13.

Reply:
https://knative.dev/docs/eventing/migration/ping/